sound/soc/intel/boards/sof_da7219.c

```diff
@@ -281,6 +281,9 @@ sof_card_dai_links_create(struct device *dev, struct snd_soc_card *card,
 			return -EINVAL;
 		}
 		break;
+	case CODEC_MAX98390:
+		max_98390_dai_link(dev, ctx->amp_link);
```

Comment:
We need to select SND_SOC_MAX98390 in config SND_SOC_INTEL_SOF_DA7219_MACH if sof_da7219 supports max98390.

Reply:
fixed. thanks for reminding.